test/gsb-test-srv/proto/safebrowsing.proto

```diff
@@ -0,0 +1,467 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Confirmed: since the original repo only exports these types via an `internal` package the `.proto` def needs to be in our src tree. I added a comment about its origins.